src/khoj/processor/conversation/utils.py

```diff
@@ -194,6 +196,7 @@ def truncate_messages(
         assert type(system_message.content) == str
         current_message = "\n".join(messages[0].content.split("\n")[:-1]) if type(messages[0].content) == str else ""
         original_question = "\n".join(messages[0].content.split("\n")[-1:]) if type(messages[0].content) == str else ""
+        original_question = f"\n{original_question}"
```

Comment:
Why add a new line before the `original_question`?

Reply:
It's getting concatenated at the end without a new line if it's not explicitly added, which is why I'm adding it in here (before truncation).